packages/angular/migrations.json

```diff
@@ -1305,6 +1341,19 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "16.7.3": {
+      "version": "16.7.3-beta.0",
+      "x-prompt": "Do you want to update the Angular version to v16.2?",
+      "requires": {
+        "@angular/core": ">=16.1.0 <16.2.0"
+      },
+      "packages": {
+        "@angular/pwa": {
+          "version": "~16.2.0",
+          "alwaysAddToPackageJson": false
+        }
+      }
```

Comment:
We can't do this. We already have a package update group for Angular 16.2. We must not duplicate the prompt. Folks already in 16.2 can get a prompt asking them again if they want to update to 16.2.

Please include this package update in that one. We already missed all these updates in previous versions. Given the lack of previous feedback, the usage should be pretty low and/or anyone who had that issue already manually updated it. Anyone in the future coming to the latest version will get the update correctly.

Reply:
That's fine, i wasn't sure on the best way to handle it for this recent version.

It makes sense to me to add it to the previous one, and we can advise to use `--from --to` syntax with migrate